functions.php

```diff
@@ -56,4 +56,22 @@ function displayPokemon(array $pokemons): string {
         }
     }
     return $result;
+}
+
+
+function addPokemonToDatabase() {
```

Comment:
For PSR2 and functions the { need to go on its own line ie
`function addPokemonToDatabase() `
`{`

Reply:
I thought that was for classes only? As all the funcitons I have written have all been written like that. As in function () {

Reply:
From PSR website
![image](https://user-images.githubusercontent.com/65677163/94708396-73c2cf00-033c-11eb-9a3f-faf370232f93.png)